samples/weather_function/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM openjdk:8u171-jdk-stretch as builder
+WORKDIR /opt
+RUN wget "https://github.com/JetBrains/kotlin-native/releases/download/v0.7.1/kotlin-native-linux-0.7.1.tar.gz"
+RUN tar -xzf kotlin-native-linux-0.7.1.tar.gz && rm kotlin-native-linux-0.7.1.tar.gz
```

Comment:
Can we extract a "variable" or something for the version?
Would be great - if we update the sample to a newer version - to just change the variable...